src/main/drivers/timer.h

```diff
@@ -107,6 +111,26 @@ typedef struct timerHardware_s {
 #endif
 } timerHardware_t;
 
+#ifdef USE_TIMER_MGMT
+
+typedef uint8_t timerTag_t;
+
+typedef struct timerChannelConfig_s {
```

Comment:
But it is fine to finish and test this PR without it, it should be only considered before merging